### PR TITLE
Fix graphs page when fastlane isn’t enabled

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: app/views/update_checks/graphs.html.erb [hunk 1]
```diff
@@ -14,7 +14,10 @@
 
 <h4>Absolute numbers 🚀</h4>
 <ul>
-<li>Total number of <i>fastlane</i> launches: <b><%= number_with_delimiter(@cumulative.find { |a| a[:label] == "fastlane" }[:data].last) %></b></li>
+<% fl_data = @cumulative.find { |a| a[:label] == "fastlane" } %>
+<% if fl_data %>
+  <li>Total number of <i>fastlane</i> launches: <b><%= number_with_delimiter(@cumulative.find { |a| a[:label] == "fastlane" }[:data].last) %></b></li>
```
Comment (milch) [inline app/views/update_checks/graphs.html.erb]: `fl_data = @cumulative.find { |a| a[:label] == "fastlane" }` is used two lines above, so:

`number_with_delimiter(@cumulative.find { |a| a[:label] == "fastlane" }[:data].last)` -> `number_with_delimiter(fl_data[:data].last)`

**Review by milch — CHANGES_REQUESTED** [review]

File: app/views/update_checks/graphs.html.erb (hunk 1 above)
Comment (KrauseFx) [reply app/views/update_checks/graphs.html.erb]: whops, that's not what I wanted

**Comment by KrauseFx** [comment]
Done now 👍

**Comment by milch** [comment]
Beautiful! 🐎

**Review by milch — APPROVED** [review]

**Pull request merged**